### PR TITLE
Improve error handling

### PR DESCRIPTION
**Pull request by robhoes**
Depends on xapi-project/xcp-idl#248.

### CODE REVIEW

File: lib/network_utils.ml [hunk 1]
```diff
@@ -369,11 +377,13 @@ module Ip = struct
...
 
   let link_set dev args =
+    Sysfs.assert_exists dev;
     ignore (call ~log:true ("link" :: "set" :: dev :: args))
 
   let link_set_mtu dev mtu =
     try ignore (link_set dev ["mtu"; string_of_int mtu])
-    with e -> error "MTU size is not supported: %s" (string_of_int mtu)
+    with Network_error (Script_error _) ->
+      error "MTU size is not supported: %s" (string_of_int mtu)
```
Comment (lindig) [inline lib/network_utils.ml]:
```
error "MTU size is not supported: %d" mtu
```

Comment (robhoes) [reply lib/network_utils.ml]: Thanks, I've updated it.

**Comment by robhoes** [comment]
The Travis test won't pass yet, because it hasn't got the idl update yet.

**Comment by robhoes** [comment]
The test is happy now (@lindig).

**Pull request merged**